### PR TITLE
Wired Lordicon Icons

### PR DESCRIPTION
**Pull request by intozen**
## Illustration Information 

Wired Lineal | `https://lordicon.com/icons/wired/lineal`

Lordicon | `https://lordicon.com/`

Screenshot | `https://user-images.githubusercontent.com/97742841/196002247-2f19dece-363f-4f40-af62-7f3ba59256bd.png
`

---

## Illustration Category

Freemium

---

- ✅ I have read the [Contribution Guidelines]

### CODE REVIEW

**Comment by Anmol-Baranwal** [comment]
@intozen 
Are you doing this contribution to Hacktoberfest 2022?

**Comment by intozen** [comment]
> @intozen Are you doing this contribution to Hacktoberfest 2022?

YES.

File: README.md [hunk 1]
```diff
@@ -289,7 +290,8 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
...
+| <a href="https://www.handz.design/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184808197-3d32a0ff-c114-46d5-9e17-2e3a688096bc.png' alt="Handz_3D_Freemium_Illustrations" /></a> | [Handz 3D](https://www.handz.design/)  <br><br> ✨ Freemium <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• PSD<br>• AI<br>• PNG<br><br> ✌️Author <br> Handz Design
+| <a href="https://lordicon.com/icons/wired/lineal"><img width='700' src='https://user-images.githubusercontent.com/97742841/196000594-b1056def-415f-4fb9-bfbb-f745234a262d.png' alt="Wired Lineal" /></a> | [Lordicon](https://lordicon.com/icons/wired/lineal)  <br><br> ✨ Freemium  <br><br>Format <br> • PNG<br>• SVG<br>• GIF<br>•JSON<br>•Embed HTML<br>•APNG<br>•MP4<br>•WebP<br><br> ✌️Author <br> Lordicon |
```
Comment (Anmol-Baranwal) [inline README.md]: - add format `EPS` & `After Effects
- change alt text to `Wired_Lineal_freemium_illustrations`
- change the title to `Wired Lordicon Icons`, in the menu as well
- Correct the alphabetical order

File: README.md [hunk 2]
```diff
@@ -289,7 +290,8 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
...
-| <a href="https://www.handz.design/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184808197-3d32a0ff-c114-46d5-9e17-2e3a688096bc.png' alt="Handz_3D_Freemium_Illustrations" /></a> | [Handz 3D](https://www.handz.design/)  <br><br> ✨ Freemium <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• PSD<br>• AI<br>• PNG<br><br> ✌️Author <br> Handz Design                                                                        |
+| <a href="https://www.handz.design/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184808197-3d32a0ff-c114-46d5-9e17-2e3a688096bc.png' alt="Handz_3D_Freemium_Illustrations" /></a> | [Handz 3D](https://www.handz.design/)  <br><br> ✨ Freemium <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• PSD<br>• AI<br>• PNG<br><br> ✌️Author <br> Handz Design
```
Comment (Anmol-Baranwal) [inline README.md]: Don't disturb anything other than the illustrations you need to add.

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
Any other repository maintainer could mark this as spam or invalid, but I will not do so since it may ruin your journey of Hacktoberfest.

Hacktoberfest is an excellent opportunity and adding a single illustration isn't up to the expectations of Hacktoberfest since it could be done in mere minutes. But adding a whole list will definitely make your contribution worthwhile for Hacktoberfest. 

So, it is recommended to work on the issues exclusively made for Hacktoberfest.

Nonetheless, make these minor changes in a single commit and I will merge this pull request.

**Comment by intozen** [comment]
done.
I did the changes you mentioned kindly check.
and I will add more illustrations.

File: README.md (hunk 2 above)
Comment (intozen) [reply README.md]: you can check now

**Review by intozen — COMMENTED** [review]
you can check mow

**Review by intozen — COMMENTED** [review]
I did the changes as you mentioned but I got a mail as pr running failed.can you please check?

**Comment by Anmol-Baranwal** [comment]
@intozen 
Never add extra illustrations than that mentioned for the pull request.

Remove the flaticon since it is not an illustration but rather an organization of icons otherwise, the list would go on, and on.

**Comment by intozen** [comment]
> @intozen Never add extra illustrations than that mentioned for the pull request.
> 
> Remove the flaticon since it is not an illustration but rather an organization of icons otherwise, the list would go on, and on.

Done.

File: README.md (hunk 1 above)
Comment (Anmol-Baranwal) [reply README.md]: @intozen 

You haven't completed the fourth point i.e. arrange it in alphabetical order.

Change the menu name too.

**Comment by Anmol-Baranwal** [comment]
closing the PR based on the unresolved conflicts (iconz was removed from the readme)

**Pull request closed without merging**